crates/astria-sequencer/src/accounts/query.rs

```diff
@@ -55,21 +55,12 @@ async fn get_trace_prefixed_account_balances<S: StateRead>(
     let stream = state
         .account_asset_balances(address)
         .map_ok(|asset_balance| async move {
-            let native_asset = state
```

Comment:
Note on this change: getting the native asset here was never necessary and unnecessarily complicates the code.